src/workerd/jsg/jsg.c++

```diff
@@ -149,6 +150,14 @@ void Lock::setLoggerCallback(kj::Function<Logger>&& logger) {
   IsolateBase::from(v8Isolate).setLoggerCallback({}, kj::mv(logger));
 }
 
+void Lock::requestGcForTesting() const {
+  if (!isPredictableModeForTest()) {
+    KJ_LOG(ERROR, "Test GC used while not in a test");
+    return;
+  }
+  IsolateBase::from(v8Isolate).requestGcForTesting();
```

Comment:
You can just do `v8Isolate->RequestGarbageCollectionForTesting(...)` here and delete the method from `IsolateBase`.